kubernetes/pipelines/deploy_uci/Jenkinsfile

```diff
@@ -0,0 +1,51 @@
+@Library('deploy-conf') _
+node() {
+    try {
+        String ANSI_GREEN = "\u001B[32m"
+        String ANSI_NORMAL = "\u001B[0m"
+        String ANSI_BOLD = "\u001B[1m"
+        String ANSI_RED = "\u001B[31m"
+        String ANSI_YELLOW = "\u001B[33m"
+
+        stage('checkout public repo') {
+            folder = new File("$WORKSPACE/.git")
+            if (folder.exists())
+            {
+               println "Found .git folder. Clearing it.."
+               sh'git clean -fxd'
+            }  
+            checkout scm
+        }
+
+        stage('deploy') {
+            envDir = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-3].trim()
+	        module = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-2].trim()
+	        jobName = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-1].trim()
+		    values = [:]
```

Comment:
Please correct spacing of this line and lines above this

Reply:
Also why is this file included here? Its already part of the UCI PR right

Reply:
No, this is part of this PR only. GQL does not have build and artifactupload jobs, It has only deploy job.
So geetha provided this jenkinsfile.